libpod/runtime_img.go

```diff
@@ -154,3 +224,25 @@ func GetInsecureRegistries() ([]string, error) {
 	}
 	return registries, nil
 }
+
+// GetStorageImage retrieves an image matching the given name or hash from
+// system storage.
+// If no matching image can be found, an error is returned.
```

Comment:
Ill hit you up on irc for clarification on this func as well.

Reply:
We only talked briefly about this, but I've removed this as you indicated elsewhere I can get the info here from the runtime (r).